dev/flipbook.tsx

```diff
@@ -133,7 +133,10 @@ function SideBySideQuestionRenderer({
                 />
                 <GradableRenderer
                     question={question}
-                    apiOptions={{...apiOptions, flags: {mafs: {segment: true}}}}
+                    apiOptions={{
+                        ...apiOptions,
+                        flags: {mafs: {segment: true, linear: true}},
```

Comment:
Oh-- I didn't realize we needed to add this here as well! May as well add `linear-system`, `polygon`, and `ray` too, right?
```suggestion
                        flags: {mafs: {segment: true, linear: true, linear-system: true, ray: true, polygon: true}},
```

Reply:
Prettier will have something to say about the whitespace